wgsl/index.bs

```diff
@@ -7131,7 +7131,6 @@ can be rewritten as
 `{ let p = &(`|e1|`); *p = *p + (`|e2|`); }`
 </blockquote>
 where the identifier `p` is chosen to be different from all other identifiers in the program.
-</p>
```

Comment:
This one looks like it was meant to close the `<p>` on 7124. It seems that one is getting auto-closed before the blockquote. Perhaps try indenting everything inside of the `<p></p>`, instead of removing the closing tag - I think that will make bikeshed understand the nesting correctly. Or change the `<p>` to a nested `<div>`.

Reply:
@kainino0x thank you very much for the review! I tried to apply the `div` suggestion. Does it look better now? Thank you!